File: Labs and Projects/Week 1/mario/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int height = 0;

//scan input
    do
    {

        printf("Height: ");
        scanf(" %d", &height);

    }
    while (height > 8 || height < 1);

    int i = 1;

//building pyramid
    for (i = 1; i <= height; i++)
    {

        //tabbing
        for (int g = i; g <= height - 1; g++)
        {

            printf(" ");

        }

        //left side
        for (int j = 0; j < i; j++)
        {

            printf("#");

        }
        //print space
        printf("  ");

        //right side
        for (int k = 0; k < i; k++)
        {

            printf("#");

        }

        printf("\n");

    }
}

File: Labs and Projects/Week 6/sentimental-mario-more/mario.py
# TODO
import cs50

height = 0

while not (height <= 8 and height >= 1):
    height = int(cs50.get_int("Height:"))

for i in range(1, height + 1):
    for g in range(i, height):
        print(" ", end="")
    for j in range(0, i):
        print("#", end="")
    print("  ", end="")
    for k in range(0, i):
        print("#", end="")
    print("\n", end="")